src/main/java/com/photolocos/enterprise/dao/PhotoDAO.java

```diff
@@ -88,7 +89,8 @@ public void saveImage(MultipartFile file) throws IOException {
     }
 
     @Override
-    public PhotoDTO fetchByLocation(int locationID) {
+    public Set<PhotoDTO> fetchByLocation(LocationDTO location) throws Exception {
```

Comment:
is it going to return more than one?
At present, it returns null.
I'm more concerned with returning null than returning a Set.